cmd/promtool/unittest.go

```diff
@@ -128,14 +128,25 @@ func (utf *unitTestFile) maxEvalTime() time.Duration {
 	return maxd
 }
 
-// resolveFilepaths joins all relative paths in a configuration
-// with a given base directory.
-func resolveFilepaths(baseDir string, utf *unitTestFile) {
+// resolveAndGlobFilepaths joins all relative paths in a configuration
+// with a given base directory and replaces all globs with matching files.
+func resolveAndGlobFilepaths(baseDir string, utf *unitTestFile) {
```

Comment:
I would make `resolveAndGlobFilepaths` return an `error` and fail if something goes wrong with `filepath.Glob()`.